pkg/apis/toolchain/v1alpha1/conditions.go

```diff
@@ -0,0 +1,34 @@
+package v1alpha1
+
+import (
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
```

Comment:
May I know why we have the api types here and not in the `codeready-toolchain/api` repo?

Reply:
We want to keep toolchain operator independent at least for now.
Toolchain by itself should not require our hosted toolchain operators. It will be installed in different environments too.
It's OK to re-use some common code from hosted toolchain though.

This is why toolchain-operator contains the API, e2e tests, etc are in the toolchain-operator repo and do not share the infra with the hosted CRT operators.